extensions/blocks/podcast-player/style.scss

```diff
@@ -2,10 +2,114 @@
  * Frontend (site) styles for Podcast Episodes
  */
 
+$episode-v-padding: 15px;
+$episode-h-padding: 10px;
+$episode-status-icon-size: 22px;
+$text-color: #92959a;
+$text-color-hover: #000;
+$text-color-active: $text-color-hover;
+$text-color-error: #db5051;
+$block-bg-color: #fff;
+$block-border-color: $text-color;
+$font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell,'Helvetica Neue', sans-serif;
+$font-size: 16px;
+
 .wp-block-jetpack-podcast-player {
-    border: 1px solid;
+	border: 1px solid $text-color;
+	background-color: $block-bg-color;
+
+	audio {
+		display: none;
+	}
+}
+
+ol.podcast-player__episodes {
```

Comment:
I think we can remove these HTML element references from these classes, like `ol.podcast-player__episodes` can just be `.podcast-player__episodes`. Same goes for `li.podcast-player__episode` and `a.podcast-player__episode-link`.